keyboards/bastardkb/skeletyl/readme.md

```diff
@@ -0,0 +1,17 @@
+# Skeletyl
+
+A very small keyboard made for ergonomic enthusiasts.
+
+* Keyboard Maintainer: [Bastard Keyboards](https://github.com/Bastardkb/)
+* Hardware Supported: elite-C V4
+* Hardware Availability: [Bastardkb.com](https://bastardkb.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make bastardkb/skeletyl:manna-harbour_miryoku
```

Comment:
```suggestion
    make bastardkb/skeletyl:default
```

Reply:
Hello,

The default keymap for the skeletyl is a dummy blank one.
It's meant to be used with layouts made for 5x3, Miryoku being the most popular one - this is why by default I recommend to compile with Miryoku layout.

Regards,

Reply:
from the PR Checklist: 

- default (and via) keymaps should be “pristine”
  - bare minimum to be used as a “clean slate” for another user to develop their own user-specific keymap
  - standard layouts preferred in these keymaps, if possible
  - default keymap should not enable VIA – the VIA integration documentation requires a keymap called via

Eg, the default should be a reasonable default.